tests/unit/test_settings/test_settings_questions.py

```diff
@@ -12,14 +12,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import pytest
 import argilla_sdk as rg
 
 
 class TestQuestions:
-    def test_label_question_init(self):
-        question = rg.LabelQuestion(name="label_question", labels=["label1", "label2"])
+    @pytest.mark.parametrize(
+        "labels",
+        [
+            ["label1", "label2", "label3"],
+            {"label1": "1", "label2": "2", "label3": "3"},
+        ],
+    )
+    def test_label_question_init(self, labels):
```

Comment:
Maybe we can have a specific test checking initialization from the labels dictionary